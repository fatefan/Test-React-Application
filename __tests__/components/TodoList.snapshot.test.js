import React, { Component } from 'react'
import TodoList from 'TodoList';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';

test('<TodoList /> SnapShot', ()=>{
    let data = [{completed: true,text:'hello world'},{completed: false,text:'eat'}];
    const onTodoClick = jest.fn();
    const wrapper = render(<TodoList  todos={data} onTodoClick={onTodoClick}  />)
    expect(toJson(wrapper)).toMatchSnapshot();
})