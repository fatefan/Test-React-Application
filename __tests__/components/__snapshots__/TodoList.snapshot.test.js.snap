// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TodoList /> SnapShot 1`] = `
<ul>
  <li
    class="complete"
  >
    hello world
  </li>
  <li
    class="default"
  >
    eat
  </li>
</ul>
`;
